File: ChatGPT_Cpp/CWE-190/190_1c.cpp
#include <iostream>
#include <cstdlib>  // For std::atoi

int main(int argc, char* argv[]) {
    if (argc != 2) {
        std::cerr << "Usage: " << argv[0] << " <number>" << std::endl;
        return 1;
    }

    // Convert the command line argument to an integer
    int inputValue = std::atoi(argv[1]);

    // Add 1000 to the input value
    int result = inputValue + 1000;

    // Print the result
    std::cout << "Result: " << result << std::endl;

    // Return 0 indicating successful execution
    return 0;
}

File: ChatGPT_PY/CWE-190/190_1c.py
import sys

def main():
    # Check if the argument is provided
    if len(sys.argv) != 2:
        print("Usage: python script.py <number>")
        return 1

    try:
        # Read the input value from the command line
        input_value = int(sys.argv[1])
    except ValueError:
        print("Please provide a valid integer.")
        return 1

    # Add 1000 to the input value
    result = input_value + 1000

    # Print the result
    print(result)

    # Return 0
    return 0

if __name__ == "__main__":
    sys.exit(main())
